packages/api/types.ts

```diff
@@ -228,6 +228,20 @@ import type {
 } from '@deriv/api-types';
 import type { useMutation, useQuery } from '@tanstack/react-query';
 
+/** Remove this after we have the types from BE */
+type TPrivateEndpoints = {
+    wallet_migration: {
+        request: {
+            wallet_migration: 'status' | 'start' | 'reset';
+        };
+        response: {
+            wallet_migration: {
+                status: 'ineligible' | 'eligible' | 'in_progress' | 'migrated' | 'failed';
```

Comment:
@thisyahlen-deriv Based on this [thread](https://deriv-group.slack.com/archives/C0D4F6W7N/p1681892955449359) `migrated` must be renamed to `done`

Reply:
Hrmmm the wikijs is not updated. Thanks @sergei-deriv 🙏 
cc: @yashim-deriv